libs/ledger-live-common/src/families/ethereum/test-dataset.ts

```diff
@@ -30,6 +30,25 @@ export const ethereum1: AccountRaw = {
   xpub: "",
 };
 
+export const ethereum2: AccountRaw = {
+  id: "js:2:ethereum:0x789d2f10826BF8f3a56Ec524359bBA4e738Af5bF:",
+  seedIdentifier: "0x789d2f10826BF8f3a56Ec524359bBA4e738Af5bF",
+  name: "Ethereum legacy xpub6Bem...JyAdpYZy",
+  derivationMode: "",
+  index: 0,
+  freshAddress: "0x789d2f10826BF8f3a56Ec524359bBA4e738Af5bF",
+  freshAddressPath: "44'/60'/0'/0/0",
+  freshAddresses: [],
+  pendingOperations: [],
+  operations: [],
+  currencyId: "ethereum",
+  unitMagnitude: 18,
+  balance: "",
+  blockHeight: 0,
+  lastSyncDate: "",
+  xpub: "",
+};
```

Comment:
can you move this directly in synchronisation.integration.test.ts ? this test-dataset.ts file is soon gone.